src/live.html

```diff
@@ -130,28 +130,34 @@ <h1 id="title">Live</h1>
 		<nav>
 			<ul>
 				<li data-bind-view="dayof">
-					<a href="#/dayof">Home</a>
+					<a href="#/dayof">Day of</a>
```

Comment:
I don't get why is this tab called `day of`.

Reply:
Day of is referred to all the info you get during the "day of" event. I wanted to try it to check if this way people would find it easier to understand.

Reply:
But the section is not showing different info for each day

Reply:
oh you changed it